csrc/dynamic_transform.h

```diff
@@ -31,12 +31,6 @@ class DynamicTransformInitialInfoBuilder;
 //! sizes
 class DynamicTransformInitialInfo {
  public:
-  bool operator==(const DynamicTransformConcretizationInfo& other) const;
```

Comment:
Shouldn't the corresponding implementation be removed as well?

Reply:
Looks like it doesn't exist. 😃

Reply:
:-D Oops!